### ISSUE TITLE
Matrix shows experiments that can't be viewed due to permissions

### ISSUE THREAD

**Issue by MichaelClerx** [ModellingWebLab/WebLab#333]
> **gary**  27 minutes ago
> But it should show some previous grey runs for the model that highlighted this, although when I click on some of those I'm getting a 'Not found' error(!) https://scrambler.cs.ox.ac.uk/experiments/3910/versions/4042

... 

> **jonathan**  < 1 minute ago
> I’m getting the ‘not found’ too, I think because I can’t see some versions of the model (it’s owned by you I believe)
>
> **jonathan** 2 minutes ago
> I can see the grey experiments where I can see the model version (e.g. https://scrambler.cs.ox.ac.uk/experiments/18537/versions/18909). So it’s a bug really in the matrix view letting me see the existence of versions I can’t access

**Comment by mirams** [comment]
Yeah, wants a bit of thinking about, if it is a model/protocol combo where you *are* allowed to see some versions of the model and protocol, then perhaps simplest to be able to see all of them?

But if we do want further fine-grained control over particular versions of models/protocols, then would we want the existence of a version to be visible to anyone who had permission to see any version of an entity? So that clicking the grey square takes you to a 'Request permission to view' sort of a page?

**Comment by jonc125** [comment]
The visibility settings are per-version - which lets you do private development then 'publish' the final checked version. And I think we want to keep hiding the earlier attempts in such cases! Most parts of the front-end successfully do this; there was clearly an oversight here.
